### PR TITLE
Use the right markdown for doc

### PR DESCRIPTION
**Pull request by MalloZup**
@monstermunchkin 

https://github.com/MalloZup/terraform-provider-libvirt/blob/trouble-mark/doc/TROUBLESHOOTING.md

### CODE REVIEW

File: doc/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -38,17 +38,20 @@ apt install libvirt-dev
 
 ## Selinux on Debian distros
 
-* On Debian distros SELinux is enforced by qemu even if it is disabled globally, this might cause unexpected `Could not open '/var/lib/libvirt/images/<FILE_NAME>': Permission denied` errors. Double check that `security_driver = "none"` is uncommented in `/etc/libvirt/qemu.conf` and issue `sudo systemctl restart libvirt-bin` to restart the daemon.
+* On Debian distros SELinux is enforced by qemu even if it is disabled globally, this might cause unexpected `Could not open '/var/lib/libvirt/images/<FILE_NAME>': Permission denied` errors.
```
Comment (monstermunchkin) [inline doc/TROUBLESHOOTING.md]: This shouldn't be a list (i.e. it shouldn't start with an `*`)

File: doc/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -38,17 +38,20 @@ apt install libvirt-dev
 
 ## Selinux on Debian distros
 
-* On Debian distros SELinux is enforced by qemu even if it is disabled globally, this might cause unexpected `Could not open '/var/lib/libvirt/images/<FILE_NAME>': Permission denied` errors. Double check that `security_driver = "none"` is uncommented in `/etc/libvirt/qemu.conf` and issue `sudo systemctl restart libvirt-bin` to restart the daemon.
+* On Debian distros SELinux is enforced by qemu even if it is disabled globally, this might cause unexpected `Could not open '/var/lib/libvirt/images/<FILE_NAME>': Permission denied` errors. 
+
+Double check that `security_driver = "none"` is uncommented in `/etc/libvirt/qemu.conf` and issue `sudo systemctl restart libvirt-bin` to restart the daemon.
 
 
 ## Cannot get terraform (building from src)
```
Comment (monstermunchkin) [inline doc/TROUBLESHOOTING.md]: IMHO the title is misleading. The issue here is that it cannot find the "context" package.

File: doc/TROUBLESHOOTING.md (hunk 1 above)
Comment (MalloZup) [reply doc/TROUBLESHOOTING.md]: Ok

File: doc/TROUBLESHOOTING.md (hunk 2 above)
Comment (MalloZup) [reply doc/TROUBLESHOOTING.md]: Ok thx!

**Comment by MalloZup** [comment]
done :lollipop:

**Pull request merged**